### DISCUSSION TITLE
tidbits/starship-a-terminal-prompt-that-puts-the-most-important-info-up-front

### DISCUSSION THREAD

**Discussion by giscus** [Sydney-Informatics-Hub/tidbits#2] [Announcements]
# tidbits/starship-a-terminal-prompt-that-puts-the-most-important-info-up-front

As part of my continuing quest to make my terminal more colourful, I've starting using Starship. Starship gives you a colourful, customisable prompt, with lots of useful information about your …

https://sydney-informatics-hub.github.io/tidbits/starship-a-terminal-prompt-that-puts-the-most-important-info-up-front.html

**Comment by hlydecker** [comment]
```bash  
 __                 
/  \        _____________ 
|  |       /             \
@  @       | It looks    |
|| ||      | like you    |
|| ||   <--| are writing |
|\_/|      | a blog.     |
\___/      \_____________/
             
```
